pkg/api/pluginproxy/ds_proxy.go

```diff
@@ -75,6 +75,37 @@ func (proxy *DataSourceProxy) HandleRequest() {
 		FlushInterval: time.Millisecond * 200,
 	}
 
+	if proxy.ds.Type == m.DS_THANOS {
+		query := proxy.ctx.Req.Request.URL.Query()
+
+		dsParams := proxy.ds.SecureJsonData.Decrypt()
```

Comment:
any reason to encrypt these options, they do not seem to contain security sensitive information.

Reply:
Nah, just secure sounded more secure :dancing_men: will do it via normal jsonData :D